assets/styles.css

```diff
@@ -0,0 +1,814 @@
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(4, 1fr);
+  max-width: 70%;
+  margin: 0 auto;
+  box-sizing: border-box;
+  justify-content: center;
+}
+
+.four-grid-container h2 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  color: red;
+}
+
+.second-headings {
+  display: flex;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  color: red;
+  justify-content: space-between;
+  margin-top: 25px;
+}
+
+.grid-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.grid-img p {
+  font-size: 0.8em;
+  color: #aaa;
+}
+
+.non-flex-section3 > h2 {
+  padding-left: 27%;
+  color: red;
+}
+
+.small-background-img h2 {
+  color: red;
+}
+
+.second-background-img *,
+.third-background-img * {
+  color: white;
+  text-decoration: none;
+}
+
+.second-headings h1 {
+  margin-right: 15px;
+  font-size: 1.3em;
+}
+
+.s-image {
+  height: 50px;
+  width: 50px;
+  position: relative;
+  top: 75px;
+  margin-left: 10px;
+}
+
+.non-flex-section {
+  display: block;
+}
+
+.spiff-background {
+  background-color: red;
+  color: white;
+}
+
+.venture-background {
+  background-color: #ffece7;
+}
+
+.non-flex-section .spiff-background,
+.non-flex-section .venture-background {
+  padding-left: 22%;
+  margin-right: 25px;
+}
+
+.spiff-background h4 {
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: 20px;
+  padding-top: 10px;
+  margin-left: 70px;
+}
+
+.venture-background p {
+  margin-top: 0;
+  margin-bottom: 10px;
+  margin-left: 70px;
+}
+
+.second-flex-div i {
+  margin-block-start: 1em;
+  margin-right: 10px;
+}
+
+.p-tag {
+  border-top: 1px solid pink;
+  font-size: 0.7em;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-div {
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fill, minmax(150px, 225px));
+  max-width: 1192px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  justify-content: center;
+}
+
+.flex-div h3 {
+  margin-right: 10px;
+}
+
+.flex h3 {
+  font-size: 0.9em;
+}
+
+.second-flex-div {
+  display: flex;
+  padding-left: 4%;
+}
+
+.flex-div-h3 {
+  padding-right: 7px;
+}
+
+.small-background-img {
+  background-position-y: 50px;
+}
+
+.small-background-img > :nth-child(1) * {
+  z-index: 2;
+  opacity: 1;
+}
+
+.small-background-img p {
+  display: block;
+  margin-block-start: 5px;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.flex-div2 {
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, 1fr);
+  max-width: 70%;
+  margin: 0 auto;
+  box-sizing: border-box;
+  justify-content: center;
+}
+
+.non-flex-section2 {
+  display: block;
+}
+
+.small-background-img ul {
+  padding-left: 14px;
+  font-size: 0.9em;
+}
+
+.small-background-img ul li {
+  border-bottom: 1px solid gray;
+  color: #aaa;
+  padding-top: 7px;
+  padding-bottom: 7px;
+}
+
+.small-background-img ul li:hover {
+  color: black;
+  cursor: pointer;
+}
+
+.last-grid-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.non-flex-section3 {
+  display: block;
+}
```

Comment:
Same style definition of the selector on line 462. Group selectors, or create a new class.